casper/src/test/scala/coop/rchain/casper/CliqueOracleTest.scala

```diff
@@ -23,61 +25,40 @@ class CliqueOracleTest
 
   implicit val logEff = new LogStub[Task]
 
+  def createBlock(bonds: Seq[Bond])(creator: ByteString)(
+      parent: BlockMessage,
+      justifications: Map[Validator, BlockMessage]
+  )(implicit store: BlockStore[Task], dagStore: IndexedBlockDagStorage[Task]): Task[BlockMessage] =
+    createBlock[Task](
+      Seq(parent.blockHash),
+      creator,
+      bonds,
+      justifications.map { case (v, bm) => (v, bm.blockHash) }
+    )
+
   // See [[/docs/casper/images/cbc-casper_ping_pong_diagram.png]]
   it should "detect finality as appropriate" in withStorage {
     implicit blockStore => implicit blockDagStorage =>
-      val v1     = generateValidator("Validator One")
-      val v2     = generateValidator("Validator Two")
-      val v1Bond = Bond(v1, 2)
-      val v2Bond = Bond(v2, 3)
-      val bonds  = Seq(v1Bond, v2Bond)
+      val v1             = generateValidator("Validator One")
+      val v2             = generateValidator("Validator Two")
+      val v1Bond         = Bond(v1, 2)
+      val v2Bond         = Bond(v2, 3)
+      val bonds          = Seq(v1Bond, v2Bond)
+      val blockCreatorV1 = createBlock(bonds)(v1) _
```

Comment:
Instead of using two functions you could have 
```scala 
def blockCreator(v) = createBlock(bonds)(v)
```

Reply:
the aim is to make it as concise as possible to make it easy to follow

Reply:
blockCreator(v1)(...) is as concise as blockCreatorV1(...), the difference being the fact that for the first one you are sure that v1 is used while on the second one it's just a naming convention

Reply:
it's not as concise. it's 2 letters longer. (used 21 times => 42 letters that carry no meaning)
IMO there is a benefit to named things and this particular case is an obvious example - name the elements of interest in pre and use them.